### ISSUE TITLE
Port allocated bug

### ISSUE THREAD

**Issue by this-kramer** [upb-uc4/hlf-network#93]
**Describe the bug**
Sometimes the following error message appears in the pipeline.
```The Service "orderer-org0" is invalid: spec.ports[0].nodePort: Invalid value: 30010: provided port is already allocated```

**Additional context**
Check if the port is commonly used since we do not use that port twice.
https://github.com/upb-uc4/hlf-network/pull/92/checks

**Comment by NikoBergemann** [comment]
occured again

The Service "peer2-org2" is invalid: spec.ports[0].nodePort: Invalid value: 30221: provided port is already allocated

**Comment by anemonekk** [comment]
@palkra the ports are not commonly used, ports 30000-32767 define the valid range for NodePort usage. Could it be that a new cluster is started too soon before an old one released all ports? Maybe we would not have the problem, if we did not specify any value for the nodePort field ourselves and allow a dynamic/ automatic allocation?

**Comment by palkra** [comment]
@anemonekk I don't think that an old instance of the network is blocking the ports since we only start the network once per test.
Not specifying the nodePorts is possible, but would add much complexity to testing since it would require us to read the port every time.

**Comment by NikoBergemann** [comment]
At least the org1-ca port has to remain static, since it is set in our hlf-api tests

or we have to read it from an env var, like kind-worker-ip

**Comment by this-kramer** [comment]
Closed because stale

**Comment by this-kramer** [comment]
This is most likely an issue with the GitHub runner. Rerunning the pipeline usually helps.
